docs/sdk-keys/target-apps.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Target Apps
+sidebar_label: Target Apps
+---
+
+Coming Soon!
+
+## Motivation
+
+There are two main attributes you can add to SDK Keys which restrict their access to certain configs or config definitions - Environments, and Target Apps.  A “Target App” is a user defined abstraction tied to entities in your Statsig Project which can be linked to one or more SDK Keys.
+Target apps should be things like your "Android" or "iOS" apps, or could be more restricted to specific services like "Search" and "Feed" which may span both client and server usage.
+
+Defining and using Target Apps is a best practice when using statsig, and has two main benefits:
```

Comment:
Is this actually a best practice we want everyone to follow, or just larger organizations? Feels like it risks adding significant debugging complexity for minimal performance gains for smaller companies/ teams.

Reply:
Still a best practice I would say.  We also need to collect all of those somewhere...sigh, ill do that next